tests/FSharp.Data.GraphQL.IntegrationTests.Server/Startup.fs

```diff
@@ -2,29 +2,41 @@
 
```

Comment:
Can we link this file from StarWars too?

Reply:
I'll link as many of these as I can on the 2023/02/LHK-607-Introspection_Query branch.